modules/apps/commerce/commerce-frontend-js/src/main/resources/META-INF/resources/components/account_selector/views/AccountCreationModalBody.js

```diff
@@ -0,0 +1,208 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+import ClayForm, {ClayInput, ClaySelect} from '@clayui/form';
+import ClayIcon from '@clayui/icon';
+import ClayModal from '@clayui/modal';
+import ClayMultiSelect from '@clayui/multi-select';
+import {fetch} from 'frontend-js-web';
+import React, {useEffect, useMemo, useState} from 'react';
+
+const ORGANIZATIONS_ROOT_ENDPOINT = '/o/headless-admin-user/v1.0/organizations';
+
+const orgUrl = new URL(
+	`${themeDisplay.getPathContext()}${ORGANIZATIONS_ROOT_ENDPOINT}`,
+	themeDisplay.getPortalURL()
+);
+
+export default function AccountCreationModalBody({
+	accountData,
+	accountTypes,
+	setAccountData,
+}) {
+	const [organizationQuery, setOrganizationQuery] = useState('');
+	const [organizationData, setOrganizationData] = useState([]);
+
+	useEffect(() => {
+		async function fetchData() {
+			await fetch(orgUrl.toString())
+				.then((response) => response.json())
+				.then((data) => setOrganizationData(data.items));
+		}
+
+		fetchData();
```

Comment:
Home come you used async/await here? Isn't it the same if you just remove that part and only have

```js
fetch(orgUrl.toString())
	.then((response) => response.json())
	.then((data) => setOrganizationData(data.items))
```